backend/src/models/user.ts

```diff
@@ -0,0 +1,26 @@
+
+export interface User extends UserDisplayData {
+    password: string;
+    tokens: string[];
+    likedApartments: string[];
+    dislikedApartments: string[];
+    googleID? : string
+    isGoogle?: boolean
+}
+
+export interface UserDisplayData {
+    email: string;
+    username: string;
+    avatarUrl: string;
+    sex: Gender;
```

Comment:
```suggestion
    gender: Gender;
```
I'd make it consistent